build.sbt

```diff
@@ -22,11 +22,19 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 
 val libName = "auth-client"
 
+lazy val root = (project in file(".")).
+  enablePlugins(BuildInfoPlugin).
+  settings(
+    buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
+    buildInfoPackage := "hello"
+  )
```

Comment:
is this part still needed?

Reply:
hmmmm scalaVersion and sbtVersion are not needed no